extensions/wikia/ArticleVideo/scripts/featured-video.attribution.js

```diff
@@ -1,12 +1,26 @@
 define('wikia.articleVideo.featuredVideo.attribution', [
 		'wikia.mustache',
 		'wikia.articleVideo.featuredVideo.templates',
-		'JSMessages'
+		'JSMessages',
+		'wikia.tracker'
 	],
-	function (mustache, templates, msg) {
+	function (mustache, templates, msg, tracker) {
 		'use strict';
 
+		var track = tracker.buildTrackingFunction({
+				category: 'featured-video',
+				action: tracker.ACTIONS.CLICK,
+				trackingMethod: 'analytics'
+			});
+
 		return function (playerInstance) {
+			// tracking for initial video attribution
+			$('.featured-video__attribution-username, .featured-video__attribution-icon').click(function() {
+				track({
+					label: playerInstance.getPlaylistItem(0).username
```

Comment:
According to https://developer.jwplayer.com/jw-player/docs/javascript-api-reference/ getPlaylistItem with no number returns current playlist item. Giving index 0 will probably return the first one.

Reply:
Ah, but in this case it's ok, since this will always be the first item :)